From 6409292ab5f7d1a60a6f8948bd6dcd8633c36dfe Mon Sep 17 00:00:00 2001
From: Randy Dunlap <rdunlap@infradead.org>
Date: Tue, 14 Jul 2026 20:07:19 -0700
Subject: [PATCH 05092/15333] soundwire: bus.h: repair kernel-doc comments

Use the correct struct names (i.e., correct typos).
Use the struct keyword for structs as required.

Prevents 5 warnings:

Warning: drivers/soundwire/bus.h:86 expecting prototype for struct
  sdw_btp_section. Prototype was for struct sdw_bpt_section instead
Warning: drivers/soundwire/bus.h:100 expecting prototype for struct
  sdw_btp_msg. Prototype was for struct sdw_bpt_msg instead
Warning: drivers/soundwire/bus.h:125 cannot understand function prototype:
  'struct sdw_port_runtime'
Warning: drivers/soundwire/bus.h:144 cannot understand function prototype:
  'struct sdw_slave_runtime'
Warning: drivers/soundwire/bus.h:165 cannot understand function prototype:
  'struct sdw_master_runtime'

Signed-off-by: Randy Dunlap <rdunlap@infradead.org>
Link: https://patch.msgid.link/20260715030719.757781-1-rdunlap@infradead.org
Signed-off-by: Vinod Koul <vkoul@kernel.org>
---
 drivers/soundwire/bus.h | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/soundwire/bus.h b/drivers/soundwire/bus.h
index 8115c64dd48e..44e4f5193917 100644
--- a/drivers/soundwire/bus.h
+++ b/drivers/soundwire/bus.h
@@ -73,7 +73,7 @@ struct sdw_msg {
 };
 
 /**
- * struct sdw_btp_section - Message section structure
+ * struct sdw_bpt_section - Message section structure
  * @addr: Start Register address accessed in the Slave
  * @len: number of bytes to transfer. More than 64Kb can be transferred
  * but a practical limit of SDW_BPT_MSG_MAX_BYTES is enforced.
@@ -87,7 +87,7 @@ struct sdw_bpt_section {
 };
 
 /**
- * struct sdw_btp_msg - Message structure
+ * struct sdw_bpt_msg - Message structure
  * @sec: Pointer to array of sections
  * @sections: Number of sections in the array
  * @dev_num: Slave device number
@@ -110,7 +110,7 @@ int sdw_find_row_index(int row);
 int sdw_find_col_index(int col);
 
 /**
- * sdw_port_runtime: Runtime port parameters for Master or Slave
+ * struct sdw_port_runtime - Runtime port parameters for Master or Slave
  *
  * @num: Port number. For audio streams, valid port number ranges from
  * [1,14]
@@ -133,7 +133,7 @@ struct sdw_port_runtime {
 };
 
 /**
- * sdw_slave_runtime: Runtime Stream parameters for Slave
+ * struct sdw_slave_runtime - Runtime Stream parameters for Slave
  *
  * @slave: Slave handle
  * @direction: Data direction for Slave
@@ -151,7 +151,7 @@ struct sdw_slave_runtime {
 };
 
 /**
- * sdw_master_runtime: Runtime stream parameters for Master
+ * struct sdw_master_runtime - Runtime stream parameters for Master
  *
  * @bus: Bus handle
  * @stream: Stream runtime handle
-- 
2.55.0

